templates/components/products/options/set-rectangle.html

```diff
@@ -6,6 +6,21 @@
             <small>{{lang 'common.required'}}</small>
         {{/if}}
     </label>
+
+    {{#unless required}}
+        <input
+            class="form-radio"
+            type="radio"
+            id="attribute_rectangle__{{../id}}_none"
+            name="attribute[{{../id}}]"
+            value=""
```

Comment:
Should it be empty? or it should be `value="{{lang 'products.none'}}"`?

Reply:
I see, but I have wrote a util function which prevents sending empty values to backend. Change its work with 'none' value will make it not reusable

Reply:
Got it, thank you for clarifying 🙇